test/test_fd_passing.py

```diff
@@ -0,0 +1,101 @@
+"""This tests the ability to send and receive file descriptors in dbus messages"""
+from dbus_next.service import ServiceInterface, method
+from dbus_next.signature import SignatureTree, Variant
+from dbus_next.aio import MessageBus
+from dbus_next import Message, MessageFlag
+import socket
+
+import pytest
+
```

Comment:
Be sure these tests are good because someone might come along and do the glib implementation and need to modify this.